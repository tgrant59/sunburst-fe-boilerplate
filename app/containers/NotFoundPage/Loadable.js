import Loadable from 'react-loadable'

import PageLoading from 'components/PageLoading/index'

export default Loadable({
    loader: () => import('./index'),
    loading: PageLoading,
})
