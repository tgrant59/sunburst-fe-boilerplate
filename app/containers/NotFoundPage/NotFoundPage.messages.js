export default {
    notFoundMessage: 'Page Not Found',
}
