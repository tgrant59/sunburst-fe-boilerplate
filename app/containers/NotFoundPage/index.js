import React from 'react'

import messages from './NotFoundPage.messages'

const NotFoundPage = () => <p>{messages.notFoundMessage}</p>

export default NotFoundPage
