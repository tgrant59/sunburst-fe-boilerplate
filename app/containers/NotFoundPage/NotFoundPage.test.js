import React from 'react'
import { shallow } from 'enzyme'

import NotFoundPage from '.'

describe('<NotFoundPage />', () => {
    it('should match shapshot', () => {
        const renderedComponent = shallow(<NotFoundPage />)
        expect(renderedComponent).toMatchSnapshot()
    })
})
