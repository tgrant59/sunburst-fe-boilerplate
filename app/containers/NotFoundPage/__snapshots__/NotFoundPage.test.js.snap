// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> should match shapshot 1`] = `
<p>
  Page Not Found
</p>
`;
